Vagrantfile

```diff
@@ -7,7 +7,7 @@ Vagrant.configure("2") do |config|
   # please see the online documentation at vagrantup.com.
 
   # Every Vagrant virtual environment requires a box to build off of.
-  config.vm.box = "Ubuntu 12.04"
```

Comment:
How is this change related to what you are doing?

Reply:
Well, we  can have this change catered in a separate commit.